velox/exec/SortWindowBuild.cpp

```diff
@@ -179,15 +200,20 @@ void SortWindowBuild::computePartitionStartRows() {
   partitionStartRows_.push_back(0);
 
   VELOX_CHECK_GT(sortedRows_.size(), 0);
-  for (auto i = 1; i < sortedRows_.size(); i++) {
-    if (partitionCompare(sortedRows_[i - 1], sortedRows_[i])) {
-      partitionStartRows_.push_back(i);
-    }
-  }
 
-  // Setting the startRow of the (last + 1) partition to be returningRows.size()
-  // to help for last partition related calculations.
-  partitionStartRows_.push_back(sortedRows_.size());
+  // Currently, computePartitionStartRows find the partition start rows is a
```

Comment:
Please just describe the forward-to-back double search algorithm. Only say its better than linear or binary search.

Reply:
Done.